From 62c26817e477cf77e6fa73d139c6771bd34c4542 Mon Sep 17 00:00:00 2001
From: ceca5556 <ceca5556@colorado.edu>
Date: Sat, 19 Oct 2024 15:00:53 -0600
Subject: [PATCH] attempting to use modprobe isntead of insmod

---
 scull/scull_load | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/scull/scull_load b/scull/scull_load
index 424e9ad..71fdf7f 100755
--- a/scull/scull_load
+++ b/scull/scull_load
@@ -14,7 +14,8 @@ fi
 # invoke insmod with all arguments we got
 # and use a pathname, as insmod doesn't look in . by default
 echo "$0"
-insmod /lib/modules/$(uname -r)/extra/$module.ko $* || exit 1
+# insmod /lib/modules/$(uname -r)/extra/$module.ko $* || exit 1
+modprobe $module.ko $* || exit 1
 
 # retrieve major number
 major=$(awk "\$2==\"$module\" {print \$1}" /proc/devices)
